lyricsbot/errors.py

```diff
@@ -0,0 +1,10 @@
+"""
+Database errors.
+"""
+
+
+class ConnectionError(Exception):  # pylint: disable=W0622
```

Comment:
Seems `psycopg2` has own errors. Even if you want custom, name it according to docstring (database connection error).

Reply:
Done.